erdpy/projects/report/options/twiggy_paths_check.py

```diff
@@ -0,0 +1,59 @@
+import logging
+from pathlib import Path
+
+from erdpy import dependencies, myprocess, utils
+from erdpy.errors import BadFile
+from erdpy.projects.report.options.report_option import ReportOption
+
+logger = logging.getLogger("projects.report.options.twiggy_paths_check")
+
+
+class TwiggyPathsCheck(ReportOption):
+    def __init__(self, name: str, pattern: str) -> None:
+        super().__init__(name)
+
+        self.pattern = pattern
+
+
+    def apply(self, wasm_path: Path) -> str:
+        twiggy_paths_path = get_twiggy_paths_path(wasm_path)
+        try:
+            text = utils.read_text_file(twiggy_paths_path)
+            return str(self.pattern in text)
+        except BadFile:
+            return 'N/A'
+    
+    def requires_twiggy_paths(self):
+        return True
+
+
+def replace_file_suffix(file_path: Path, suffix: str) -> Path:
```

Comment:
If private, perhaps mark it as private? (and others, in general, when applicable)